apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V37__Create_Schema_Idiomatic_Names.sql

```diff
@@ -0,0 +1,1787 @@
+    meta_dual_elgbl_stus_nov_cd              character varying(2),                     -- medicaidDualEligibilityNovCode
+    meta_dual_elgbl_stus_dec_cd              character varying(2),                     -- medicaidDualEligibilityDecCode
+    cst_shr_grp_jan_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupJanCode
+    cst_shr_grp_feb_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupFebCode
+    cst_shr_grp_mar_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupMarCode
+    cst_shr_grp_apr_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupAprCode
+    cst_shr_grp_may_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupMayCode
+    cst_shr_grp_jun_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupJunCode
+    cst_shr_grp_jul_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupJulCode
+    cst_shr_grp_aug_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupAugCode
+    cst_shr_grp_sept_cd                      character varying(2),                     -- partDLowIncomeCostShareGroupSeptCode
+    cst_shr_grp_oct_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupOctCode
+    cst_shr_grp_nov_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupNovCode
+    cst_shr_grp_dec_cd                       character varying(2),                     -- partDLowIncomeCostShareGroupDecCode
+    drvd_line_1_adr                          character varying(40),                    -- derivedMailingAddress1
+    drvd_line_2_adr                          character varying(40),                    -- derivedMailingAddress2
+    drvd_line_3_adr                          character varying(40),                    -- derivedMailingAddress3
+    drvd_line_4_adr                          character varying(40),                    -- derivedMailingAddress4
+    drvd_line_5_adr                          character varying(40),                    -- derivedMailingAddress5
+    drvd_line_6_adr                          character varying(40),                    -- derivedMailingAddress6
+    city_name                                character varying(100),                   -- derivedCityName
+    state_cd                                 character varying(2),                     -- derivedStateCode
+    state_cnty_zip_cd                        character varying(9),                     -- derivedZipCode
+    bene_link_key                            bigint,                                   -- beneLinkKey
+    constraint beneficiaries_pkey
+        primary key (bene_id)
+);
+
+create index if not exists beneficiaries_hicn_idx
```

Comment:
for consistency should this be called beneficiaries_bene_crnt_hic_num_idx?

Reply:
did some renaming to try to adhere to your suggestion; where we run into issue(s) is that the index and/or constraint names are limited to 64 characters. Thus, we will always have a case where the name can't quite 'fit' into 64 chars; for example:
`
create index if not exists beneficiary_monthly_yearmonth_partdcontract_parentbeneficiary_idx
	on beneficiary_monthly (year_month, partd_contract_number_id asc, parent_beneficiary asc);
`

will get truncated to 64 chars. Now I am open to any/all suggestions as to how these things could be named, but at the end of the day, it's just a property name that needs to be unique within the schema.